tests/real_tests.rs

```diff
@@ -61,9 +62,16 @@ impl Drop for RealTestDev {
     }
 }
 
-/// Set up count devices from configuration file.
-/// Wipe first MiB on each device.
-fn get_devices(count: u8) -> Option<Vec<RealTestDev>> {
+#[allow(dead_code)]
+enum DeviceLimits {
+    Exactly(usize),
+    AtLeast(usize),
+    Range(usize, usize), // inclusive
+}
+
+/// Return one or more lists of devices to use, based upon the
+/// constraints. Returns None if constraints can't be met.
+fn get_devices(limits: DeviceLimits) -> Option<Vec<Vec<String>>> {
```

Comment:
It would be nicer if ```get_devices()``` result type were ```Option<Vec<Vec<PathBuf>>>``` and ```RealTestDev::new()``` took a &Path as an argument, see: #554.